Unified/closor.py

```diff
@@ -322,6 +324,26 @@ def rank( wfn ):
             to = ['hn-cms-relval@cern.ch']
             sendEmail(subject, text, destination=to )
             ## just announced ; take it out now.
+            #BI.pop( bname )
+            subject = "Low Statistics for Relval"
+            issues = ""
+            if batch_extreme_warnings[ bname ]:
+                issues="The following datasets have outstanding completion (<50%%) issues:\n\n"
+                issues+="\n".join( sorted( batch_extreme_warnings[ bname ] ))
+                issues+="\n\n"
+                text = ""
+                text+= "Dear all,\n\n"
+                text+= "A batch of release validation workflows has finished.\n\n"
+                text+= "Batch ID:\n\n"
+                text+= "%s\n\n"%( bname )
+                text+= "Detail of the workflows\n\n"
+                text+= "https://dmytro.web.cern.ch/dmytro/cmsprodmon/requests.php?campaign=%s\n\n"%( bname )
+                text+= "%s\n\n"%(issues) 
+                text+= "This is an automated message.\n\n"
+                text+= ""
+                to = ['hn-cms-relval@cern.ch']
+                sendEmail(subject, text, destination=to )
```

Comment:
I would prefer to send 1 email only. Normal relval completion email, and in case of low stat, send one low stat email only.

Reply:
So I would remove the first email path (the unformatted block) and use `if` statement to modify some part of the subject and body, and send the email. Makes the code much leaner.

Reply:
Not really remove, but merge the 2 parts.

Reply:
But, I thought what we discussed in PnR meeting was to send an additional email. If you say, we change it to one email only.

Reply:
yeah, I mean the older email format is pre-existent. I didn't touch that, just added the new one. If we want, we can change that too in a better format.